lua/cfc_notifications/shared/sh_base.lua

```diff
@@ -32,6 +32,11 @@ if SERVER then
     hook.Add( "PlayerDisconnected", "CFC_NotificationUnready", function( ply )
         CFCNotifications.playersReady[ply] = true
     end )
+
+    -- Give chance for other addons to load
+    timer.Simple( 0, function()
```

Comment:
may as well make this 1 second

Reply:
fair

Reply:
Actually, client side initialisation runs on the Initialise hook, gonna use that instead